tools/topology/topology2/platform/intel/sdw-amp-generic.conf

```diff
@@ -420,14 +420,36 @@ IncludeByKey.NUM_SDW_AMP_LINKS {
 					num_input_pins 1
 					Object.Base.input_audio_format [
 						{
-							in_bit_depth            32
-							in_valid_bit_depth      32
+							IncludeByKey.SDW_AMP_NUM_CHANNELS {
```

Comment:
Question for @bardliao on how ```NUM_SDW_AMP_LINKS=3``` should be handled

Reply:
Also question if feedback alh-copiers and pipelines need more channels for xover tplg variant build, or do they already contain all channels from all amplifiers.